app/templates/components/repository-list.hbs

```diff
@@ -8,6 +8,17 @@
         (eq hasRequiredContacts true)
       )
     }}
+      {{#if
+        (eq (await (prefixes-available model.provider.id)) 0)
+      }}
+        <div class="panel panel-transparent">
+          <div class="panel-body" cy-data="alert">
+              <BsAlert @dismissible={{false}} @type="warning">
+                Contact Support to get more prefixes for new repositories.
```

Comment:
@svogt0511 maybe I'm mistaken but I don't think there is a case in which this scenario will happen. Can you help me up understand this message? 

If prefixes are always automatically assigned... Why would anybody would need to be warned and this?

Reply:
Looks like that should have been taken out.  I think you wanted all of those warning sremoved.  It would only show up when there are 0 prefixes available.  However, as you specified, the  'Add Repository' button is disabled when there are 0 prefixes available.

Will remove it now.

Reply:
@kjgarza - Deployed.  Removed the alert.

@kjgarza and @richardhallett - New deployment available at vercel, [here](https://bracco-onrceplze-datacite.vercel.app/).